USER.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE "user" (
+  "id" SERIAL PRIMARY KEY,
```

Comment:
Why did we go with serial?
I prefer to stay homogeneous (ie int)

Reply:
SERIAL is int, but auto generated